api-module-library/microsoft-teams/models/credential.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const mongoose = require('mongoose');
+const { Credential: Parent } = require('@friggframework/module-plugin');
+
+const schema = new mongoose.Schema({
+    graph_access_token: { type: String, trim: true, lhEncrypt: true },
```

Comment:
There's no refresh_token for the graph API access? (Is this a delegated permissions thing?)